database/migrations/2023_01_09_111415_add_users_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->boolean("is_admin");
+            $table->boolean("is_active");
+            $table->string("username");
+            $table->dropColumn("name");
+            $table->string("first_name");
+            $table->string("last_name");
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('name');
+            $table->dropColumn(['is_admin', 'is_active', 'username', 'first_name', 'lastname']);
```

Comment:
```suggestion
            $table->dropColumn(['is_admin', 'is_active', 'username', 'first_name', 'last_name']);
```
cột `lastname` ở đâu vậy bạn, mình thấy bên trên bạn sử dụng `last_name` chạy ở local mượt rùi đẩy lên nhé